pinc/wordcheck_engine.inc

```diff
@@ -1103,15 +1100,7 @@ function get_langcode3s_for_languages($languages)
 // docs (see online docs for usort()) and manual confirmation
 function deterministicStrnatcasecmp($a, $b)
 {
-    $natcaseresult = strnatcasecmp($a, $b);
-    if($natcaseresult != 0)
-    {
-        return $natcaseresult;
-    }
-    else
-    {
-        return strnatcmp($a, $b);
-    }
+    return strnatcasecmp($a, $b)?:strnatcmp($a, $b);
```

Comment:
I like the explicit check to 0 that we had before instead of relying on it being falsy.

Reply:
OK. Perhaps I'm too used to false-y 0 in C and Python.